### PR TITLE
Enable google monitoring and gitlab runners using dind 

### PR DESCRIPTION
**Pull request by Langstra**
I dont know why, but github didn't show changes I pushed to the branch. Check here the original comments. https://github.com/craigbehnke/terraform-google-gitlab-docker-autoscaler-runner/pull/7

### CODE REVIEW

**Comment by craigbehnke** [comment]
One more thing on the configuration: the typedef for the root level `var.runners.host_metric_interval` config should be optional, defaulting to 60 (`optional(number,60)`). That way there is no breaking change (I just tried to redeploy with the latest configuration and it failed on me if I didn't edit my config).

File: variables.tf [hunk 1]
```diff
@@ -54,8 +54,12 @@ variable "runners" {
     enable_display = optional(bool, false)
     # Enable integrity monitoring for the runner (default: true) See also: https://cloud.google.com/compute/shielded-vm/docs/shielded-vm#integrity-monitoring
     enable_integrity_monitoring = optional(bool, true)
+    # Enable the ops agent to view host metrics in gcp
+    enable_ops_agent = optional(bool, false)
     # The number of instances to keep on standby
-    idle_count = number
+    idle_count = optional(number, 60)
```
Comment (craigbehnke) [inline variables.tf:60]: This is very dangerous.
e2-highcpu-8 cost = $56.68304/month (VM used in example)
$56.68304 * 60 = $3400.9824/month by default

Please revert this back to `number`.

File: variables.tf [hunk 2]
```diff
@@ -54,8 +54,12 @@ variable "runners" {
...
+    # The interval at which the host metrics are reported (works together with enable_ops_agent set to true)
+    host_metric_interval = number
```
Comment (craigbehnke) [inline variables.tf:62]: Please make the type here for `host_metric_interval` = `optional(number,60)`.

**Review by craigbehnke — CHANGES_REQUESTED** [review]
Please move the `optional(number,60)` type to the `host_metric_interval` variable.

File: variables.tf (hunk 1 above)
Comment (Langstra) [reply variables.tf]: oh wow, I messed this up big time. I reverted this and added it to the host_metric_interval.

**Review by craigbehnke — APPROVED** [review]

**Comment by craigbehnke** [comment]
Thank you very much, @Langstra!

**Pull request merged**